sql/insert.go

```diff
@@ -88,6 +95,22 @@ func (p *planner) Insert(n *parser.Insert) (planNode, error) {
 				b.Put(key, string(t))
```

Comment:
I think the benefit on INSERT/UPDATE outweighs the added complexity to SELECT. Mind creating an issue for this if you don't plan on doing it immediately?